web/samples/CMakeLists.txt

```diff
@@ -199,7 +199,7 @@ foreach (NAME ${HTML_FILES})
     list(APPEND DEMO_ASSETS ${SERVER_DIR}/${NAME})
 endforeach()
 
-set(TEXTURE_FILES floor_ao_roughness_metallic.png floor_basecolor.png floor_normal.png)
+set(TEXTURE_FILES floor_ao_roughness_metallic.png floor_basecolor.jpg floor_normal.png)
```

Comment:
Do we still need the `floor_basecolor.png` file in our repo?

Reply:
Good catch, removed the useless copy.  (we still have a lossless parquet texture in in third_party)